.env

```diff
@@ -22,6 +22,8 @@ IDAM_KEY_BULK_SCAN_PROCESSOR=AAAAAAAAAAAAAAAA
 IDAM_KEY_BULK_SCAN_ORCHESTRATOR=AAAAAAAAAAAAAAAA
 IDAM_KEY_XUI_WEBAPP=AAAAAAAAAAAAAAAA
 IDAM_KEY_AM_ROLE_ASSIGNMENT=AAAAAAAAAAAAAAAA
+IDAM_KEY_TS_TRANSLATION_SERVICE=AAAAAAAAAAAAAAAC
```

Comment:
Can we keep the `IDAM_KEY_TS_TRANSLATION_SERVICE` value matching between here and hmcts/ts-translation-service#6.

Reply:
done

Reply:
done